src/StdUtils.sol

```diff
@@ -36,10 +36,24 @@ abstract contract StdUtils {
     }
 
     function bound(uint256 x, uint256 min, uint256 max) internal view virtual returns (uint256 result) {
+        require(min <= max, "StdUtils bound(uint256,uint256,uint256): Max is less than min.");
         result = _bound(x, min, max);
         console2.log("Bound Result", result);
     }
 
+    function boundInt(int256 x, int256 min, int256 max) internal view virtual returns (int256 result) {
```

Comment:
Let's just call this `bound`
```suggestion
    function bound(int256 x, int256 min, int256 max) internal view virtual returns (int256 result) {
```

Reply:
I wanted to... But Solidity says it is duplicated with `bound(uint256,uint256,uint256)` 😭.

Reply:
We can call it `bound`, because it has a different function signature than the other `bound`, so that's an allowed overload.

But if you change the name to `bound` with no other changes, solc will error because certain statements like `assertEq(bound(5, 0, 4), 0), 0)` are now ambiguous—are those inputs ints or uints? It can't tell which `bound` implementation to use.

So we can resolve that by just casting the first input value so it knows which overload to use: `assertEq(bound(uint256(5), 0, 4), 0)`

This is a bit clunky for tests where we hardcode constants and need to cast everywhere now, but I think that's ok. In standard fuzz tests, solidity will know the type of that first input to bound because it a function input of uint256 or int256 in the fuzz test, so the casting won't be needed and UX will be better for users

Reply:
Oh, I understand. Now it works well.
Applied at here. [25ff698](https://github.com/foundry-rs/forge-std/pull/253/commits/25ff69883e47be0c10b0e10fdbf7e391efc97acd)